tests/summ_ai_api/summ_ai_test.py

```diff
@@ -222,3 +232,142 @@ async def test_summ_ai_error_handling(
             "could not be automatically translated into Easy German."
             in response.content.decode("utf-8")
         )
+
+
+def test_check_translated_text_is_in_response():
+    """
+    Test for check_translated_text_is_in_response method.
+    Tests if return is True, when 'translated_text' is found in request and exception is thrown when not.
+    """
+    response_data_translation_succeeded = {
+        "dummy": "dummy",
+        "translated_text": "Das ist einfaches Deutsch.",
+    }
+    assert (
+        SummAiApiClient(
+            MockedRequest(), "form_dummy"
+        ).check_translated_text_is_in_response(response_data_translation_succeeded, 200)
+        is True
+    ), "if translated_text found in reponse-data, check_translated_text_is_in_response should return True"
```

Comment:
This is intended to test `SummAiApiClient.check_translated_text_is_in_response()`. Currently it *is* mostly a stub that seems obvious and slightly unnecessary. For philosophy about why this might still be sensible in the long run I suggest asking Ingo or me separately from this PR, I have a feeling this will become quite the extensive discussion.

Reply:
Don't get me wrong - philosophically we are 100% on the same page. I would love to see us get such test coverage in our CMS!
However, since we have unlimited potential for improvement but limited resources, I think this is a utopia we will never achieve. That doesn't mean we shouldn't try - but I think we should carefully choose the order in which we do things. That's also why we spend so much time prioritizing issues - because we know we can't solve all problems at once, we do the things first that will have the biggest impact. To apply this principle to testing, I would suggest that we write tests for complex functions that are most likely to break first, and only then write tests for trivial functions (which may or may not become complex in the future).